mmseg/datasets/transforms/loading.py

```diff
@@ -2,6 +2,7 @@
 import warnings
 from typing import Dict, Optional, Union
 
+from osgeo import gdal
```

Comment:
Since many users may not use this library, this library should not be forced to be installed by users.